pkg/kubelet/kuberuntime/security_context.go

```diff
@@ -35,14 +35,7 @@ func (m *kubeGenericRuntimeManager) determineEffectiveSecurityContext(pod *v1.Po
 		}
 	}
 
-	// TODO: Deprecated, remove after we switch to Seccomp field
-	// set SeccompProfilePath.
```

Comment:
`getSeccompProfilePath` the method is also to be removed

Reply:
already removed

Reply:
done